jasmine/src/inverted-index.js

```diff
@@ -126,13 +126,9 @@ function Index() {
 		for (var i = 0; i < terms.length; i++) {
 
 			if (!this.invertedIndex.hasOwnProperty(terms[i])) {
-				results.push(-1);
+				results.push([-1]);
 			} else {
-				if (this.invertedIndex[terms[i]].length === 1) {
-					results.push(this.invertedIndex[terms[i]][0]);
-				} else {
-					results.push(this.invertedIndex[terms[i]]);
-				}
```

Comment:
**Code quality - Nitpicks**
Changed code to require that the search function returns an array of arrays.